### ISSUE TITLE
/Packages page 404: page not found

### ISSUE THREAD

**Issue by haingoctu** [NuGet/NuGetGallery#3140]
A 404.4 page not found when you click on packages is normal? It's unclear to me what I am suppose to do. 

According to the error, the error's most likely cause is the requested URL does not have a handler configured to process the request on the Web

**Comment by maartenba** [comment]
Where are you seeing this error?

**Comment by haingoctu** [comment]
When I try to open the packages page.
![image](https://cloud.githubusercontent.com/assets/14064416/16925162/edbe8da0-4cf0-11e6-8c0d-b12cf265fea6.png)

**Comment by maartenba** [comment]
Make sure that you can run ASP.NET MVC in this server. IIS may have better logs available.

**Comment by haingoctu** [comment]
I have ASP.NET 3.5 and 4.5 installed as roles on the server. How may i get IIS logs? Are you talking about httperr1.log?

**Comment by maartenba** [comment]
You will need the latest .NET version on the server

**Comment by haingoctu** [comment]
It looks like I do have the latest .net version (4.6.1).  I can go to the home page, upload a package, register an account so shouldn't the packages page work?

**Comment by maartenba** [comment]
It should. Can you access the Admin URL? (need to add yourself as an administrator first)
There will be an "Error logs" entry in there which may be of help.

**Comment by haingoctu** [comment]
I made a test account as an admin by inserting the respective user and key(1) into the useroles table. Then I attempted to go to localhost/elmah.axd. I get a 404.4 aswell. I got that http://localhost/elmah.axd link from the wiki guide. 

Is there another admin url?

**Comment by maartenba** [comment]
That's the one. Looks like you may be missing some handler mappings in IIS (or have a module that conflicts). Can you try another machine? Searh results for 404.4 error with MVC also yield some results that are worth checking.

**Comment by haingoctu** [comment]
It turns out my inexperienced with MVC and Visual studio was the culpurit. This is all too confusing for me. I'm learning though.

I changed the Properties of NuGetGallery->Web[Servers] to Local IIS (which is what im using and not express), put my project url to localhost and created a virtual directory. On top of that I also did this:
http://stackoverflow.com/questions/18506826/iis-express-cannot-add-duplicate-collection-entry-of-type-add-with-unique-key (directed physical path to empty). I'm not sure if the latter helped but it was something I did that resulted in a success.

I'm closing it. Last question though. Where are the packages people upload located?
